Remove Beta mentions in README + Docs (#2801)
psf/black: docs/index.rst

Try it out now using the `Black Playground <https://black.vercel.app>`_.

.. admonition:: Note - Black is now stable!

   *Black* is `successfully used <https://github.com/psf/black#used-by>`_ by
   many projects, small and big. *Black* has a comprehensive test suite, with efficient
   parallel tests, our own auto formatting and parallel Continuous Integration runner.
   Now that we have become stable, you should not expect large formatting to changes in

>    the future. Stylistic changes will mostly be mostly responses to bug reports and new
>    syntax in the cpython runtime.

Reviewer: ```suggestion
   the future. Stylistic changes will mostly be responses to bug reports and support for new Python
   syntax.
```
